spyder/widgets/fileswitcher.py

```diff
@@ -488,6 +491,9 @@ def previous_row(self):
 
     def next_row(self):
         """Select next row in list widget."""
+        if self.mode == self.SYMBOL_MODE:
+            self.select_row(+1)
+            return
```

Comment:
This is needed since I found a new bug that doesn't allow you to move with the symbol finder. Since is the same element, I thought this could be included in this PR, should I open an issue with a different PR?